src/plugins/saved_objects_management/public/management_section/mount_section.tsx

```diff
@@ -57,6 +61,8 @@ export const mountManagementSection = async ({
     return children! as React.ReactElement;
   };
 
+  const spacesApi = (spacesOss?.isSpacesAvailable && spacesOss) || undefined;
```

Comment:
NIT: I would go with `spacesOss?.isSpacesAvailable ? spacesOss : undefined`, but may just be personal preference.